www/night.scss

```diff
@@ -5,6 +5,8 @@
 	.text-color-blue{fill:#5392D5;}
 	.text-size-medium{font-size:48px;}
 	.text-size-large{font-size:72px;}
+	.text-size-extra-large{font-size:144px;}
+	.text-size-extra-extra-large{font-size:216px;}
```

Comment:
I don't actually think it's necessary to specify this twice, since nightmode will only override, and not completely replace, daymode CSS. But since we are already duplicating all the other CSS perhaps this could go in a different PR